### ISSUE TITLE
[FEAT] Sequence of Commands with a Single Input

### ISSUE THREAD

**Issue by collink2451** [collink2451/ATCTrainer-IssueTracking#469]
### What is your new feature idea or request?

Working on some new Ground Files catered towards CRC. We have some scenarios where the trainee is just working CD/FD, and we tell our trainers to taxi out departures once the trainee successful issues an IFR clearance to simulate a ground & local controller online "working" the traffic.

Wondering if there could be a way for the trainer to input one command that would:
1) Wait a specified time to pushback
2) Wait a specified time to taxi
3) Wait a specified time to takeoff

Basically reducing trainer workload while also simulating realism with aircraft moving around the airport with minimal input from the trainer. The commands could "activated" somehow perhaps? (idrk im not smart like that).
